projects/ui-components/lf-selection-list/lf-selection-list.component.ts

```diff
@@ -105,20 +109,33 @@ export class LfSelectionListComponent {
       (event.shiftKey && (event.key === 'ArrowUp' || event.key === 'ArrowDown'))
     ) {
       if (event.key === 'Enter') {
-        this.onDblClick(item.value);
         this.selectable.onItemClicked(event, item, this.listItems);
+        this.itemSelected.emit({ selected: item, selectedItems: this.selectable.selectedItems });
```

Comment:
do we call this line twice since it is also called in onDblclick?

Reply:
yes, the event from lf-repository-browser is only emitted once though